docs/en/Multi-Lingual-Entities.md

```diff
@@ -0,0 +1,100 @@
+# Multi Lingual Entities
+
+ABP Framework defines two basic interfaces for Multi-Lingual entity definitions to provide a standard model for translating entities.
+
+## IHasMultiLingual
+
+`IHasMultiLingual<TTranslation>` interface is used to mark multi lingual entities. The entities marked with `IHasMultiLingual<TTranslation>` interface must define language-neutral information. The entities marked with `IHasMultiLingual<TTranslation>` contains a collection of Translations which contains language-dependent information.
+
+Example:
+
+```csharp
+public class Product : Entity, IMultiLingualEntity<ProductTranslation>
```

Comment:
@maliming I think that the interface here should be **`IHasMultiLingual`** NOT `IMultiLingualEntity`, isn't it?

Reply:
hi 
https://github.com/abpframework/abp/issues/1754#issuecomment-826184860